packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.html

```diff
@@ -33,6 +33,13 @@
         <pack-migration-building-details></pack-migration-building-details>
     </div>
 
+    <!-- ONT SHIPPING STEP -->
+    <div
+        data-ng-if="!$ctrl.loading.init && $ctrl.process.currentStep === 'ontShipping'"
```

Comment:
```suggestion
    <!-- ONT SHIPPING STEP -->
    <div data-ng-if="!$ctrl.loading.init">
    <div
        data-ng-if="$ctrl.process.currentStep === 'ontShipping'"
```
You can use only one div for each step for avoid to duplicate the code data-ng-if="!$ctrl.loading.init" for each step